db/utils.go

```diff
@@ -97,6 +99,31 @@ func InitSQLDB(port int, host, user, password, name string) (*sqlx.DB, error) {
 	return db, nil
 }
 
+// InitTestSQLDB opens test PostgreSQL database
+func InitTestSQLDB() (*sqlx.DB, error) {
+	host := os.Getenv("PGHOST")
+	if host == "" {
+		host = "localhost"
+	}
+	port, _ := strconv.Atoi(os.Getenv("PGPORT"))
+	if port == 0 {
+		port = 5432
+	}
+	user := os.Getenv("PGUSER")
+	if user == "" {
+		user = "hermez"
+	}
+	pass := os.Getenv("PGPASSWORD")
+	if pass == "" {
+		panic("No PGPASSWORD envvar specified")
+	}
+	dbname := os.Getenv("PGDATABASE")
+	if dbname == "" {
+		dbname = "hermez"
+	}
```

Comment:
There are the redundancy default variables in the makefile. Maybe make sense we use it only in one place

https://github.com/hermeznetwork/hermez-node/pull/733/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R23-R28

Reply:
I tried to make it easy to use with and without Makefile.

Reply:
IMHO, this part should remain.

Reply:
If you are doing it by code, you already cover both situations. The Makefile is redundant. I think only one check is necessary.

Reply:
We need a port, password, username, and so on in the Makefile for the docker container too.
Only PGHOST is really redundant here.

Of course, we can hardcode all docker parameters, but I like things like this:
```
export PGPORT=15432
make run-database-container
make test-unit
make stop-database-container
```
It allows us to start temporary postgresql instance on a nonstandard port and do tests with it. This could be useful if you have main postgresql instance on the 5432 port and  want to avoid any data changes in it.

Reply:
It's up to you because it's a suggestion, but less code is better for readability and maintenance. If we change the default variables, we need to remember to change it into the code and also makefile... we must remember it

Reply:
@stunndard I think it's a good idea not depend all time of Makefile or externally change some variable just for one specific test that you will not use anymore and clearly you *do not want to change the default variables*

Reply:
and also, it's a good to try avoid hardcoded variables